modules/fedimint-walletv2-common/src/config.rs

```diff
@@ -0,0 +1,261 @@
+    /// | 9         | 344  | 548     |
+    /// | 10        | 352  | 565     |
+    /// | 11        | 379  | 618     |
+    /// | 12        | 406  | 672     |
+    /// | 13        | 414  | 689     |
+    /// | 14        | 441  | 742     |
+    /// | 15        | 468  | 796     |
+    /// | 16        | 476  | 813     |
+    /// | 17        | 503  | 867     |
+    /// | 18        | 530  | 920     |
+    /// | 19        | 539  | 937     |
+    /// | 20        | 565  | 991     |
+    pub fn new(
+        bitcoin_pks: BTreeMap<PeerId, PublicKey>,
+        fee_consensus: FeeConsensus,
+        network: Network,
+    ) -> Self {
+        let tx_overhead_weight = 4 * 4 // nVersion
+            + 1 // SegWit marker
+            + 1 // SegWit flag
+            + 4 // up to 2 inputs
+            + 4 // up to 2 outputs
+            + 4 * 4; // nLockTime
+
+        let change_witness_weight = descriptor(&bitcoin_pks, &42_u64.consensus_hash())
+            .max_weight_to_satisfy()
+            .expect("Cannot satisfy the change descriptor.")
+            .to_wu();
+
+        let change_input_weight = 32 * 4 // txid
+            + 4 * 4 // vout
+            + 4 // Script length
+            + 4 * 4 // nSequence
+            + change_witness_weight;
+
+        let change_output_weight = 8 * 4 // nValue
+            + 4 // scriptPubKey length
+            + 34 * 4; // scriptPubKey
+
+        let destination_output_weight = 8 * 4 // nValue
+            + 4 // scriptPubKey length
+            + 34 * 4; // scriptPubKey
+
+        Self {
+            bitcoin_pks,
+            finality_delay: 6,
+            send_tx_vbytes: weight_to_vbytes(
+                tx_overhead_weight
+                    + change_input_weight
+                    + change_output_weight
+                    + destination_output_weight,
+            ),
+            receive_tx_vbytes: weight_to_vbytes(
+                tx_overhead_weight
+                    + change_input_weight
+                    + change_input_weight
+                    + change_output_weight,
+            ),
+            dust_limit: bitcoin::Amount::from_sat(10_000),
+            fee_consensus,
+            network,
+        }
+    }
+}
+
+#[derive(Debug, Clone, Eq, PartialEq, Hash, Serialize, Deserialize, Encodable, Decodable)]
+pub struct FeeConsensus {
+    pub base: Amount,
+    pub parts_per_million: u64,
+}
+
+impl FeeConsensus {
+    /// The mint module will charge a non-configurable base fee of one thousand
+    /// satoshis per transaction input and output to account for the costs
+    /// incurred by the federation for processing the transaction. On top of
+    /// that the federation may charge a additional relative fee per input and
+    /// output of up to ten thousands parts per million which equals one
+    /// percent.
+    ///
+    /// # Errors
+    /// - This constructor returns an error if the relative fee is in excess of
+    ///   one thousand parts per million.
```

Comment:
ten thousand?

Reply:
Could we express/docuemnt in sats/vB? dpc has a too small brain to convert on the fly.